scripts/tableGenerator.js

```diff
@@ -1578,25 +1578,20 @@ function generateCart() {
         message += `${name} : ${item.quantity}Kg \n`;
     });
     if (currentOrders.length === 0) {
-        window.alert('There are no orders placed');
+        window.confirm('There are no orders placed');
```

Comment:
Rename this `generateCart()` function to getOrderMessage(), which should only return the message as a string if there are any orders. Handle the confirm/alert box in the onClick handler(`sendOrder()`) for whatsapp button.

Doing like this will give the function a specific functionality.

Reply:
Doing that will make the code long, and about the specific functionality, I think this function has the functionality to return the confirm message.